frontend/components/Links.tsx

```diff
@@ -21,69 +47,32 @@ const Links = (props: any): ReactElement => {
         setSelectedRoute(Route.Home);
         break;
       }
-      case '/about': {
-        setSelectedRoute(Route.About);
+      case '/services': {
+        setSelectedRoute(Route.Services);
         break;
       }
-      case '/syllabus': {
-        setSelectedRoute(Route.Syllabus);
+      case '/team': {
+        setSelectedRoute(Route.Team);
         break;
       }
       case '/blog': {
         setSelectedRoute(Route.Blog);
         break;
       }
+      case '/events': {
+        setSelectedRoute(Route.Events);
+        break;
+      }
     }
   }, [router.route]);
 
   return (
     <>
-      <li>
-        <Link href='/'>
-          <a
-            className={`link underline-offset-2 ${
-              selectedRoute === Route.Home ? 'decoration-2' : 'link-hover'
-            }`}
-          >
-            Home
-          </a>
-        </Link>
-      </li>
-      <li>
-        <Link href='/about'>
-          <a
-            className={`link underline-offset-2 ${
-              selectedRoute === Route.About ? 'decoration-2' : 'link-hover'
-            }`}
-          >
-            About
-          </a>
-        </Link>
-      </li>
-      <li>
-        <Link href='/syllabus'>
-          <a
-            className={`link underline-offset-2 ${
-              selectedRoute === Route.Syllabus ? 'decoration-2' : 'link-hover'
-            }`}
-          >
-            Syllabus
-          </a>
-        </Link>
-      </li>
-      <li>
-        <Link href='/blog'>
-          <a
-            className={`link underline-offset-2 ${
-              selectedRoute === Route.Blog ? 'decoration-2' : 'link-hover'
-            }`}
-          >
-            Blog
-          </a>
-        </Link>
-      </li>
+      {Object.values(Route).filter((v) => isNaN(Number(v))).map((route, i) => (
```

Comment:
Great refactor here, removes quite a bit of redundancy

Reply:
Thanks to Lance.